relayer-engine/src/storage/storage.test.ts

```diff
@@ -59,18 +59,17 @@ describe("Workflow lifecycle happy path tests using InMemory store", () => {
   });
 
   it("gets next workflow", async () => {
-    const res = await storage.getNextWorkflow();
+    const res = await storage.getNextWorkflow(2);
     await expect(res).toBeTruthy();
     expect(res!.plugin.demoteInProgress).toBe(plugin.demoteInProgress);
     expect(res!.workflow).toStrictEqual(workflow);
-    expect(await store.hGet("__activeWorkflows", key)).toBe("1");
```

Comment:
tiny nit: this value assertion does matter. Just ensures the value we think is being inserted actually does